src/app/features/reporting/reporting/select-report/select-report.component.spec.ts

```diff
@@ -31,4 +31,62 @@ describe("SelectReportComponent", () => {
 
     expect(component.selectedReport).toBe(report);
```

Comment:
add test that daterange is displayed for non-sql reports (ideally also only if they have `?` parameters)